README.md

```diff
@@ -1 +1,160 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Awesome books <a name="about-project"></a>
+
+In this project, we will build a basic website that allows users to add/remove books from a list. We achieved that by using JavaScript objects and arrays. We needed to dynamically modify the DOM and added basic events. We will be using them in all Microverse projects and most likely in my future job as well, so it is important to know them!
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-html">HTML</a></li>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-javascript">JAVASCRIPT</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Follow a correct Gitflow**
+- **Comply with linters**
+- **DOM events with Javascript**
+- **Javascript events**
+- **Use objects to store and access data**
+- **Use localStorage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://carlosigreda.github.io)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+
+✅ Github account <br>
+✅ Visual Studio Code installed <br>
+✅ Node.js installed <br>
+✅ Git Bash installed (optional)   
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd [my-folder]
+  git clone git@github.com:CarlosIgreda/Awesome-Books.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="author"></a>
+**Carlos Igreda**
+- GitHub: [@CarlosIgreda](https://github.com/CarlosIgreda)
+- Twitter: [@carlosigreda](https://twitter.com/carlosigreda)
+- LinkedIn: [carlos-jesús-igreda-rivas-323763116](https://www.linkedin.com/in/carlos-jes%C3%BAs-igreda-rivas-323763116/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Medium-fidelity wireframe**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+## 👤 Collaborators:
+
+**Damian Dib Bustros**
+
+- GitHub: [@DamianDib9](https://github.com/DamianDib9)
+- LinkedIn: [@DamianDib9](https://ar.linkedin.com/in/damian-dib-bustros-953577261?original_referer=https%3A%2F%2Fwww.google.com%2F)
+
+**Adeyemi Olusola**
+
+- GitHub: [@Emarkees](https://github.com/emarkees)
+- LinkedIn: [@Emarkees](https://www.linkedin.com/in/adeyemi-olusola-1a3ab1141)
```

Comment:
**[OPTIONAL]**

- [ ] Kindly put these collaborators in your Authors section because these are authors of your project as well :+1:

Reply:
Done @HAMMAS-SALEEM